documentation/Thermal simulations getting started.md

```diff
@@ -2,6 +2,19 @@
 - The thermal simulations themselves are based on the output of the acoustic simulations.
 	- So running the thermal simulations without any acoustic output to base it on is not an option.
 
+Before feeding in parameters for the heating simulations it is important to note that these simulations don't model the individual pulses and their pulse-repetition frequencies (PRF) but concatenate the pulses over a trail into one block of stimulation and one block without.
+
+# Example
+Let's take a PRF of 5Hz, a stim_duration of 1s, an iti of 15s, 30 trials and a duty_cycle of 30% (0.3).
+When running the stimulation, these are the pressure changes the transducer will produce.
+- 40ms of stimulation, 160ms without stimulation.
+- This cycle is repeated 5 times (5Hz) within your stim_duration of 1s
+- After this, no stimulation will be administered for 14s.
+- This cycle will be repeated 30 times.
+
+For the simulations, the number of pressure changes is reduced to limit the computational load.
+With these same parameters, PRESTUS combines the separate 40ms periods of stimulation into one block of 200ms of stimulation and one block of 800ms without stimulation per trial. This reduces the amount of blocks per trial from 10 to 2.
```

Comment:
I'm not sure if this is correct. It will use the same 200 ms cycle (60 ms on, 140 ms off) , and will repeat it 5 times. If you set 'equal_steps' to 1, it will use the time steps defined by sim_time_steps, so if it's 10 ms, it will do 6 on steps and 14 off steps. If you set 'equal steps' to 0, it will do one on step and one off step each cycle

Reply:
Than I understood it incorrectly. When you say each cycle in the last sentence, do you mean each ms or something else?